sphinx/writers/latex.py

```diff
@@ -1378,19 +1360,25 @@ def depart_hlistcol(self, node):
         pass
 
     def latex_image_length(self, width_str):
-        match = re.match('(\d*\.?\d*)\s*(\S*)', width_str)
+        return self.latex_dim_to_latexdim(width_str)
+
+    def latex_dim_to_latexdim(self, length_str):
```

Comment:
The package name `sphinx.writers.latex` already means the classes and functions are used for LaTeX writer. I think such prefix are redundant. So I renamed it to `rstdim_to_latexdim` simply.